ext/B/B.xs

```diff
@@ -2305,6 +2307,14 @@ PadnamePV(pn)
 	SvUTF8_on(TARG);
 	XPUSHTARG;
 
+bool
+PadnameIsUndef(padn)
+       B::PADNAME      padn
+    CODE:
+        RETVAL = padn == &PL_padname_undef ? TRUE : FALSE;
```

Comment:
Why the ternary? This can simply be

```
RETVAL = padn == &PL_padname_undef;
```

Reply:
true we can simplify this, this was more a style to make it clear but I'm fine adjusting it to the short version

Reply:
changed to use the short version as part of b98236f7